sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -227,10 +295,18 @@ def test_hermite():
     assert hermite(n, -x) == (-1)**n*hermite(n, x)
     assert hermite(-n, x) == hermite(-n, x)
 
+    assert hermite(n, 0) == 2**n*sqrt(pi)/gamma(S(1)/2 - n/2)
+    assert hermite(n, oo) == oo
+
     assert conjugate(hermite(n, x)) == hermite(n, conjugate(x))
 
+    assert  str(hermite(n, x).rewrite("polynomial")) == 'factorial(n)*Sum((-1'\
+        ')**_k*(2*x)**(-2*_k + n)/(factorial(_k)*factorial(-2*_k + n)), (_k, '\
+        '0, floor(n/2)))'
```

Comment:
Could be dummy_eq

Reply:
Ah, my eye-balling wasn't that careful it seems... Will fix it.